src/gstools/covmodel/base.py

```diff
@@ -100,9 +102,21 @@ class CovModel:
         :math:`2\sin(\alpha/2)`, where :math:`\alpha` is the great-circle
         distance, which is equal to the spatial distance of two points in 3D.
         As a consequence, `dim` will be set to `3` and anisotropy will be
-        disabled. `rescale` can be set to e.g. earth's radius,
+        disabled. `geo_scale` can be set to e.g. earth's radius,
         to have a meaningful `len_scale` parameter.
         Default: False
+    geo_scale : :class:`float`, optional
+        Geographic scaling in case of latlon coordinates to get a meaningful length
+        scale. By default, len_scale is assumed to be in radians with latlon=True.
+        Can be set to :any:`EARTH_RADIUS` to have len_scale in km or
+        :any:`DEGREE_SCALE` to have len_scale in degree.
```

Comment:
I find this confusing. By just looking at it, I would assume that the `len_scale` is multiples of `geo_scale` or something like that (as `EARTH_RADIUS` is only one possible radius). Maybe reformulate that the physical unit is km as soon as a radius is set.

Reply:
I actually meant the formulation of the documentation, but also only using `KM_SCALE` is a good idea :+1:

Reply:
Done